src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,149 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService register;
+    private static User defaultUser;
+
+    @BeforeAll
+    static void beforeAll() {
+        register = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void validUserInitialization() {
+        defaultUser = new User();
+        defaultUser.setAge(19);
+        defaultUser.setLogin("first");
+        defaultUser.setPassword("1234567890");
+    }
+
+    @AfterEach
+    void afterEach() {
+        Storage.people.clear();
+        defaultUser = null;
+    }
+
+    @Test
+    void register_ValidUser_Ok() {
+        try {
+            register.register(defaultUser);
+        } catch (RuntimeException e) {
+            fail("Failed registration");
+        }
```

Comment:
```suggestion
        } catch (Exception e) {
            fail("Failed registration");
        }
```

lets make test failed message more informative